[AArch64] Remove unused references to MVT::f80.

davemgreen opened the pull request:
These f80 fp types are only supported on X86 and can be removed from AArch64. It looks like they were copied from another backend by mistake.

nasherm reviewed (approved):
LGTM

nasherm reviewed (commented):
LGTM. Obviously test failures need addressing

davemgreen committed:
[AArch64] Remove unused references to frem f80.

The f80 fp types are only supported on X86 and can be removed from AArch64.
It looks like they were copied from another backend by mistake. There is
another reference to f80 for EXTLOAD, but that is apparently used for
legalizing constants to the correct types.
